src/Stratis.Bitcoin.Features.PoA/Voting/IdleFederationMembersKicker.cs

```diff
@@ -92,17 +94,21 @@ public void Initialize()
             else
             {
                 this.logger.LogDebug("No saved data found. Initializing federation data with current timestamp.");
+                InitializeFederationMemberLastActiveTime(((PoAConsensusOptions)this.network.Consensus.Options).GenesisFederationMembers);
+            }
+        }
 
-                this.fedPubKeysByLastActiveTime = new ConcurrentDictionary<PubKey, uint>();
-
-                // If this is a clean sync initialize each member's last active time with the genesis timestamp.
-                // Each member's last active time will be updated on each block connected event during syncing
-                // or on block mined.
-                foreach (IFederationMember federationMember in ((PoAConsensusOptions)this.network.Consensus.Options).GenesisFederationMembers)
-                    this.fedPubKeysByLastActiveTime[federationMember.PubKey] = this.network.GenesisTime;
+        /// <inheritdoc />
```

Comment:
Is there any documentation for this to inherit? The removed comment looks like a decent candidate, unless the intention of the code has changed a lot

Reply:
I've added this back to the interface.